src/sentry/api/serializers/models/event.py

```diff
@@ -376,3 +376,36 @@ def serialize(self, obj, attrs, user):
             # Needed to generate minidump links in UI
             "crashFile": attrs["crash_file"],
         }
+
+
+class ExternalEventSerializer(EventSerializer):
+    """
+    Event serializer for the minimum event data needed to send to an external service. This
+    can be used for Integrations and the Integration Platform. If the full event is needed,
+    use `event.as_dict()` instead.
+    """
+
+    def serialize(self, obj, attrs, user):
+        tags = [{"key": key.split("sentry:", 1)[-1], "value": value} for key, value in obj.tags]
+        for tag in tags:
+            query = convert_user_tag_to_query(tag["key"], tag["value"])
+            if query:
+                tag["query"] = query
+
+        user = obj.get_minimal_user()
+
+        return {
+            "group_id": six.text_type(obj.group_id) if obj.group_id else None,
```

Comment:
We don't use snake case for the other APIs, any reason for this naming?

Reply:
we use snake case for the webhooks we send for the integration platform and seems like plugins may do that too? - my thinking is that since this is not used by our UI we don't need to use camel casing.

Reply:
It's not that our UI needs camel casing per se but I was under the impression that's the convention we've committed to for our public API. I'll leave that to your team to decide if webhooks / integration platform should follow a different convention.

Reply:
Consistency would be nice.

@MeredithAnya do you know if we always use snake case for webhooks? Or are we inconsistent there?

Reply:
I thought we always used camelCase for the API interfaces. Where exactly is that not happening? I think that might have been an oversight. My understanding is the same as @lynnagara's.

Reply:
Note that some of these names are changing (used to be `event_id` and didn't have any group id for example), not sure if that's going to affect anyone that's expecting this response to follow a specific schema